README.md

```diff
@@ -125,5 +164,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is the outline for the table.  It states what will go in each column and
+whether they are null, integers or characters. A one to many relationship is represented
+by crows feet. For instance if there is one mom who has many children, she will have
```

Comment:
"Crows feet" is used to indicate the "many" end of a relationship in an ERD, not schema.

We would use a "foreign" key to indicate a one-to-many relationship in a schema (e.g., `artist_id` in a `songs` table).